<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeViewLeft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="treeViewRight">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select right dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCopyFiles">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>===Copy==&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectLeftDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select left dir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
